kasa/smart/smartchilddevice.py

```diff
@@ -41,7 +41,6 @@ async def create(cls, parent: SmartDevice, child_info, child_components):
         """Create a child device based on device info and component listing."""
         child: SmartChildDevice = cls(parent, child_info, child_components)
         await child._initialize_modules()
-        await child._initialize_features()
```

Comment:
The changes to how module updatse are handled deserve their own, separate PR.

Reply:
https://github.com/python-kasa/python-kasa/pull/921